lightning/src/ln/channel.rs

```diff
@@ -5761,12 +5780,16 @@ impl<Signer: Sign> Writeable for Channel<Signer> {
 		// Note that this field is ignored by 0.0.99+ as the TLV Optional variant is used instead.
 		self.minimum_depth.unwrap_or(0).write(writer)?;
 
+		let mut htlc_maximum_msat = None;
 		match &self.counterparty_forwarding_info {
 			Some(info) => {
 				1u8.write(writer)?;
 				info.fee_base_msat.write(writer)?;
 				info.fee_proportional_millionths.write(writer)?;
 				info.cltv_expiry_delta.write(writer)?;
+				// Note that this field is ignored by 0.0.99+ as the TLV Optional variant is used instead.
```

Comment:
I'm not really sure what's going on here anymore. We shouldn't be adding new stuff.

Reply:
Ok thanks! Would it be more correct to only add this as a TLV Optional field instead?

Reply:
Yes, we can only add new fields via TLV now.